src/GraphControl/Models/KeyGraphFeaturesInfo.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "Utils.h"
+#include <string>
```

Comment:
Here and in the rest of the files add standard headers to the pch instead

Reply:
cleaned!